posts/app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :user
+
+  def ratio
```

Comment:
Não seria melhor guardar na BD o valor calculado?

Reply:
Será que essa é a melhor abordagem pensando em performance?
De modo geral a listagem acontece muito mais vezes que os votos, certo?

Reply:
Eu imaginei a listagem por ranking como uma feature adicional de uma aplicação (como "top posts of all time" no Reddit, por exemplo), não como algo tão central. Mas nesse caso salvar o ratio em BD seria uma abordagem melhor, mesmo.